packages/lesswrong/lib/collections/revisions/schema.ts

```diff
@@ -26,16 +26,13 @@ export const ContentType = new SimpleSchema({
 
 // defining a custom scalar seems to allow it to pass through any data type,
 // but this doesn't seem much more permissive than ContentType was originally
-addGraphQLSchema(`
+export const graphqlTypeDefs = gql`
```

Comment:
If you're importing both copies of this one I'd expect that to cause an error, but maybe they get deduplicated?